crypto/internal.h

```diff
@@ -115,6 +115,8 @@
 #include <openssl/stack.h>
 #include <openssl/thread.h>
 
+#include "fipsmodule/rand/snapsafe_detect.h"
```

Comment:
what's this needed for?

Reply:
I think it was so that gtest main could call the HAZMAT function.